app/src/test/java/sw19/moring03/paint/DrawingObjectManagerTest.java

```diff
@@ -0,0 +1,73 @@
+package sw19.moring03.paint;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+import sw19.moring03.paint.tools.LineTool;
+import sw19.moring03.paint.tools.PathTool;
+import sw19.moring03.paint.tools.PointTool;
+import sw19.moring03.paint.tools.Tools;
+import sw19.moring03.paint.utils.DrawingObjectManager;
+import sw19.moring03.paint.utils.PointF;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DrawingObjectManagerTest {
+    private DrawingObjectManager drawingObjectManager;
+
+    @Before
+    public void setUp() {
+        drawingObjectManager = new DrawingObjectManager();
+    }
+
+    @Test
+    public void testObjectsToRedoList() {
+        List<Tools> expectedToolList = Arrays.asList(new PointTool(), new LineTool(), new PathTool());
+        for (Tools tool : expectedToolList) {
```

Comment:
the for takes exactly the same amount of lines as if you would add the tools seperately. since loops in such tests not so nice pls split in three add stmts